### DISCUSSION TITLE
Docker installation failed

### DISCUSSION THREAD

**Discussion by SAMLCSE** [jef/streetmerchant#1169] [Help]
### Description
Docker installation failed on Rasberry Pi. Any solution please? 

```
pi@raspberrypi:~/Downloads/streetmerchant $ docker run --cap-add=SYS_ADMIN \
>   -it --rm --env-file ./.env \
>   ghcr.io/jef/streetmerchant:nightly
docker: Error response from daemon: failed to create endpoint happy_tesla on network bridge: failed to add the host (veth6965a44) <=> sandbox (veth73c9328) pair interfaces: operation not supported.
```

**Comment by jef** [comment]
Good question. I know some other folks had to download some dependencies with the RPi. Take a look at previous issues. I'm really unsure with this one.

**Comment by Burito55** [comment]
I encountered this problem as well. In the end, i skipped running docker and uses PM2 instead.

**Comment by jef** [comment]
> PM2

Oh... Now this is pretty cool. Kind of related: #922
